ctapipe/io/simteleventsource.py

```diff
@@ -413,7 +415,11 @@ def _generate_events(self):
             data.simulation.tel.clear()
             data.trigger.tel.clear()
 
-            event_id = array_event.get("event_id", -1)
+            event_id = array_event.get("event_id", 0)
+            if event_id == 0:
```

Comment:
Is there no chance that there is an `event_id=0` that is a real event?  Maybe better to return None as default?

Reply:
event_id 0 seems exclusively used for calibration events. All real shower events in simtel array are 100 x shower + reuse, so they start at 100. The extract_calibevents start at 1.